src/resources/views/register.blade.php

```diff
@@ -4,15 +4,26 @@
 @section('page_header', trans('teamspeak::register'))
 
 @section('left')
-    <div class="panel panel-default">
-        <div class="panel-heading">
-            <h3 class="panel-title">Register my Teamspeak User</h3>
+    <div class="box box-primary">
+        <div class="box-header with-border">
+            <h3 class="box-title">Register my Teamspeak User</h3>
         </div>
-        <div class="panel-body">
+        <div class="box-body">
             <p>Log into the Teamspeak server with your nickname set to the EXACT SAME as your Main character's name.</p>
-            <p>TS Name should be: <b>{{ setting('main_character_name') }}</b><p>
-            <input type="text" id="ts3id" name="ts3id" value="" maxlength="29" size="29" disabled="true" class="loading">
-            <input type="button" id="ts3register" name="ts3register" value="Click to find your name and register">
+            <p>TS Name should be: <b>
+            @if ($tags != '')
+              [{{ $ticker }}] 
+            @endif 
+            {{ setting('main_character_name') }}</b><p>
+            <p>@if ($tags != '')
+            * Note: There is a space between the corp tag and your username.
```

Comment:
You can add that inside a `<p class="help-block"></p>

But that's only an ui recommendation

Reply:
Seems github eat a bit about this one.
Surround the sentend with `p` node and Append it a class `help-block`